proto/toit/model/organization.proto

```diff
@@ -39,10 +39,14 @@ message PaymentDetails {
 }
 
 message User {
+    message Settings {
+        bool welcome_open = 1;
```

Comment:
`welcome_closed`, the default value will be `false` since it initially should be open so that is better.